tox.ini

```diff
@@ -119,13 +125,15 @@ commands=
 basepython=python3.5
 deps =
     pytest==3.0.2
+    pytest-xdist==1.15.0
 commands=
     python -m pytest tests/pytest tests/cover/test_testdecorators.py
 
 [testenv:pytest28]
 basepython=python3.5
 deps =
     pytest==2.8.2
+    pytest-xdist==1.15.0
```

Comment:
Is there any way we can cut down repetition of dependency definitions in this file? 🤔